docs/how-to/integrate-platform.md

```diff
@@ -0,0 +1,65 @@
+# Integrate PMM with Percona Platform
+Percona Platform Portal brings together database distributions, support expertise, services, management, and automated insights. 
+
+Connect your PMM servers to Percona Platform to boost the monitoring capabilities of your PMM installations and manage database deployments easier. In addition, you get access to PMM updates, automated insights, advanced advisor checks and more alert rule templates.
+
+### Connect PMM to Percona Platform
+You can connect to Percona Platform with a Percona Account or via Google or GitHub authentication. If [Percona Support](https://www.percona.com/about-percona/contact) has enabled a custom identity provider for your account, you can also log in using your company's credentials.
+
+We recommend that you connect with a Percona Account, as this gives you access to other Percona services, including Percona Platform, Percona Customer Portal, and Community Forum. If you don’t have a Percona Account, you can create one on the [Percona Platform homepage](https://portal.percona.com/login) using the **Don't have an account? Create one?** link.
+
+#### Pre-requisites
+To ensure that PMM can establish a connection to Percona Platform: 
+
+### Upgrade to PMM 2.27.0 or later
+ Before connecting your PMM server to Percona Platform, make sure you are using PMM version 2.27 or newer. Otherwise, upgrade your PMM installation beforehand. 
+ 
+ This is required because, starting with PMM 2.27, Percona Platform has replaced username/passoword authentication with access token authentication. Access-token authentication increases security and enables federated identity. 
+
+  This change did not affect existing connections to PMM Platform, which were not automatically terminated. 
+ 
+ For more information, see [Install and set up PMM](../setting-up/index.md).
+
+### Check that you are a member of an existing Platform organization
+1. Log in to [Percona Platform](https://portal.percona.com) using your Percona Account. If you are connecting via GitHub, make sure that you  set your email address as **public** in your GitHub account. If your email address is private instead, Percona Platform cannot access it to authenticate you.
+
+2. On the **Getting Started** page, check that the **Create organization** step shows an option to view your organization.
+
+If you see an option to create a new organization instead, your Percona Account is not linked to any organization yet. If this is the case, contact your account administrator, or create a new organization for your Percona Account.
+
+### Set the public address of your PMM server 
+1. In PMM, go to **Settings > Advanced Settings**.
+2. Enter your address/hostname or click **Get from browser** to enable your browser to automatically detect and populate this field.
```

Comment:
```suggestion
2. Enter your address/hostname or click **Get from browser** so that your browser automatically detects and populates this field.
```
Avoid split infinites...